src/pages/products/components/FilterSection.jsx

```diff
@@ -74,6 +55,10 @@ function FilterSection({ filterDrawer, setFilterDrawer }) {
 							});
 						}}
 					/>
+					<span className="pt-0-5" style={{ textAlign: "center" }}>
+						Products within Rs.{" "}
```

Comment:
**suggestion**: Using `{" "}` is not a good practice, try wrapping this line in a span and give padding according to the use.